Project.toml

```
@@ -0,0 +1,21 @@
+name = "MLBase"
+uuid = "f0e99cf1-93fa-52ec-9ecc-5026115318e0"
+version = "0.9.0"
+
+[deps]
+IterTools = "c8e1da08-722c-5040-9ed9-7db0dc04731e"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
+StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
+
+[compat]
+IterTools = "1.3"
+Reexport = "1.2"
```

```suggestion
IterTools = "1"
Reexport = "1"
```
I imagine v1.0 and v1.1 of those packages would also work. For StatsBase, I'm not sure what an appropriate bound is; the most recent might be similarly too restrictive. Do the tests pass for you with the latest StatsBase version? Looks like CI isn't running, I guess because this PR is adding CI configuration from a fork.

---

Yep I tested with v1.0 for those two.
It runs fine with StatsBase v0.33.10 (latest). Its v0.33.1 is a year old, so I think setting its compat at "0.33" is fine.

---

I didn't realize 0.33 was that old. That's fine then.